draft-irtf-cfrg-vdaf.md

```diff
@@ -4603,8 +4626,15 @@ perhaps unlikely) for a large set of non-heavy-hitter values to share a common
 prefix, which would be leaked by a prefix tree with a sufficiently small
 threshold.
 
-The only known, general-purpose approach to mitigating this leakage is via
-differential privacy.
+A malicious adversary controlling the Collector and one of the Aggregators can
+further turn arbitrary non-heavy prefixes into heavy ones by tampering with the
+DPF output at any position. While our construction ensures that the sum of all
+nodes evaluated at a level cannot exceed the value at the parent nodes, this
+still may allow an adversary to discover individual non-heavy strings.
+
+The only practical, general-purpose approach to mitigating these leakages is via
+differential privacy, which is RECOMMENDED for all protocols using Poplar1 for
```

Comment:
Should this become a MUST?

Reply:
I don't think we can make it a MUST without specifying a mechanism for achieving it.

Reply:
I don't think that's true. For instance, OHTTP doesn't specify how clients obtain the configuration of gateways, but still has a MUST about it ([source](https://datatracker.ietf.org/doc/html/rfc9458#section-7)):

> A specific method for a Client to acquire configurations is not included in this specification. Applications using this design MUST provide accommodations to mitigate tracking using Client configurations.

So I think we could include a MUST about using DP, since the VDAF's privacy guarantees don't hold unless you do.

Reply:
I'm fine with it being a MUST.

Reply:
I'm leaving it at RECOMMENDED for now, since I'm not sure prescribing DP but not any particular DP parameters would be meaningful. In particular, implementers could satisfy the requirement by implementing DP with $\delta=1$ or similar insecure settings.